# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ladspa-swh-plugins/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- swh-plugins-0.4.12/util/pitchscale.c_orig	2005-10-22 21:20:43.049712568 +0200
+++ swh-plugins-0.4.12/util/pitchscale.c	2005-10-22 21:21:08.974771360 +0200
@@ -41,7 +41,7 @@
 
 #include "pitchscale.h"
 
-static float ps_in[MAX_FRAME_LENGTH*2], ps_out[MAX_FRAME_LENGTH*2];
+float ps_in[MAX_FRAME_LENGTH*2], ps_out[MAX_FRAME_LENGTH*2];
 static fft_plan aplan = NULL, splan = NULL;
 
 void pitch_scale(sbuffers *buffers, const double pitchScale, const long
